CHANGELOG.md

```diff
@@ -1,3 +1,21 @@
+## 15.5.4 (April 11, 2017)
+
+* Fix compatibility with Enzyme by exposing `batchedUpdates` on shallow renderer. [See #9382](https://github.com/facebook/react/commit/69933e25c37cf5453a9ef132177241203ee8d2fd).
```

Comment:
Sure - maybe that will be less confusing after all. Although then it's a bit tricky to map the add-on version to the React version. I guess I would annotate them in the section where React was using that version of the add-on.